src/main/java/com/corundumstudio/socketio/namespace/Namespace.java

```diff
@@ -265,7 +249,16 @@ public BroadcastOperations getRoomOperations(String room) {
         return new SingleRoomBroadcastOperations(getName(), room, getRoomClients(room), storeFactory);
     }
 
-    @Override
+	@Override
+	public BroadcastOperations getRoomsOperations(String... rooms) {
```

Comment:
please fix a typo - roomOperations

Reply:
@mrniko I renamed these methods to `getMultipleRoomOperations` otherwise the method below will recurses infinitely 
<img width="1006" alt="image" src="https://github.com/mrniko/netty-socketio/assets/70547118/3e2111b2-7dbc-48db-bee7-da66717b94f5">

<img width="580" alt="image" src="https://github.com/mrniko/netty-socketio/assets/70547118/08becf72-5320-4196-9db5-b2b0deeb33e0">

Reply:
@mrniko Done

Reply:
Example use case: 
<img width="902" alt="image" src="https://github.com/mrniko/netty-socketio/assets/70547118/13c19eb6-855b-4af2-9a1b-b40e9c731f88">
@mrniko will you merge?